codegen.py

```diff
@@ -190,6 +191,10 @@ def noop_(x): return None
   attr {term} value = unsafeAttribute {{ key: "{x}", value: cb' (Cb (const (value $> true))) }}''')
             print_(f'''instance Attr anything {term} (Effect Boolean) where
   attr {term} value = unsafeAttribute {{ key: "{x}", value: cb' (Cb (const value)) }}''')
+            print_(f'''instance Attr anything {term} (Zora Unit) where
+  attr {term} value = unsafeAttribute {{ key: "{x}", value: cb' (Cb (const (runImpure (value $> true)))) }}''')
+            print_(f'''instance Attr anything {term} (Zora Boolean) where
+  attr {term} value = unsafeAttribute {{ key: "{x}", value: cb' (Cb (const (runImpure value))) }}''')
```

Comment:
This was added because `FRP.Event` defines `bus` as the following:
```purs
bus :: forall m a b. ((a -> m Unit) -> AnEvent m a -> b) -> AnEvent m b
```
While `Deku.Core` defines `bus` such that it performs lifting automatically.
```purs
bus :: forall a b. ((a -> Effect Unit) -> AnEvent Zora a -> b) -> AnEvent Zora b
```